tests/Makefile

```
@@ -0,0 +1,12 @@
+.PHONY: run-stack stop-stack load-data
+
+export VERSION=8.4.0-SNAPSHOT
+
+run-stack:
+	docker-compose up -d
+
+stop-stack:
+	docker-compose down --volumes
+
+load-data:
+	../loadsample.sh
```

```suggestion
	./loadsample.sh
```

Is it a typo here @tarekziade?